lib/elixir/src/elixir_utils.erl

```diff
@@ -33,6 +33,14 @@ get_line(Opts) when is_list(Opts) ->
     false -> 0
   end.
 
+get_generated(Opts) when is_list(Opts) ->
+  case erlang:system_info(otp_release) of
+    Release when Release >= "19" ->
```

Comment:
Just to make sure I have a clear understanding of the implications here, this means that a .beam compiled on 19 will produce dialyzer warnings on 18, and a .beam compiled on 18 will still exhibit the issue which produced this patch (on both 18 and 19), but have no other effect?